mod_reforged/hooks/entity/tactical/enemies/ghost.nut

```diff
@@ -4,7 +4,7 @@
 		this.actor.onInit();
 		this.setRenderCallbackEnabled(true);
 		local b = this.m.BaseProperties;
-		b.setValues(this.Const.Tactical.Actor.Ghost);
+		b.setValues(::Const.Tactical.Actor.Ghost);
```

Comment:
Don't do these specific `this.Const` to `::Const` changes in this commit (i.e. lines 7, 27, 28). You can discard these ones completely, we'll do them in a later cleanup commit in the codebase. Only fix the `::Const` in the new code you added.